src/main/java/com/github/princesslana/greedorama/Portfolio.java

```diff
@@ -86,6 +86,10 @@ public MonetaryAmount getCash() {
         .collect(Collectors.toList());
   }
 
+  public List<Transaction> getTransactions() {
+    return List.copyOf(transactions);
```

Comment:
Since the list is immutable we don't need to make a defensive copy. We can just return transactions.

Reply:
Okay, I'll do that.